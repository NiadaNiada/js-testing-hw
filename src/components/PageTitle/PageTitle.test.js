import React from 'react';
import PageTitle from "./PageTitle";
import { render } from "@testing-library/react";

describe('<PageTitle />', function () {
  it('should match the snapshot', () => {
    const { container: { firstChild } } = render(<PageTitle label={'Github user details'} />);
    expect(firstChild).toMatchSnapshot();
  });
  /**
   * Write a snapshot test for PageTitle component
   * Don't forget to add snapshot file to the git
   */
});
