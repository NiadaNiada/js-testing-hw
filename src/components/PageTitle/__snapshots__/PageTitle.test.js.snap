// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageTitle /> should match the snapshot 1`] = `
<div>
  <h1
    class="page-title"
    data-testid="page-title"
  >
    Github user details
  </h1>
</div>
`;
